### DISCUSSION TITLE
Wish: Focus follows file...

### DISCUSSION THREAD

**Discussion by Keks-Dose** [jarun/nnn#1932] [Q&A]
I'm opening files in nnn by hitting -> (right arrow, same effect with hitting enter). A PDF opens in Okular. But nnn / kitty keeps the focus. I'd like to customize nnn that an app opened out of nnn automatically gets the focus.

**Comment by N-R-K** [comment]
Shouldn't the DE/WM auto focus on newly opened window? What WM are you using? Check the config, there's likely a switch for it.

**Comment by Keks-Dose** [comment]
Linux, OpenSUSE, Tumbleweed, up-to-date. And no, this isn't the behavior of the WM: KDE Plasma. If I open a PDF out of the file manager (dolphin), the newly opened Okular gets the focus.

**Comment by N-R-K** [comment]
> If I open a PDF out of the file manager (dolphin), the newly opened Okular gets the focus.

nnn is a terminal file manager. It doesn't interact with or try to change WM's focus or whatnot. Look into either configuring your DE/WM or maybe use a custom opener (see `NNN_OPENER` docs) which also switches focus with [`xdo`](https://github.com/baskerville/xdo) or some other utility.
